Modules/Utils.cs

```diff
@@ -1862,6 +1858,11 @@ public static void ApplySuffix(PlayerControl player)
                     _ => name
                 };
             }
+            
+            if (!name.Contains('\r') && player.FriendCode.GetDevUser().HasTag())
```

Comment:
Why `&& !Main.AutoMuteUs.Value`  was removed?

Reply:
It’s because I changed the code before it was added